### ISSUE TITLE
shanghaitech_b evaluation

### ISSUE THREAD

**Issue by adrian-dalessandro** [cvlab-stonybrook/DM-Count#31]
I'm trying to understand the evaluation procedure that you have used for the ShanghaiTech B dataset. Are you monitoring performance on the test set during training and then selecting the best performance? Or, otherwise, how are you deciding when to stop training to achieve the best test performance? ShanghaitechA/B don't have validation sets given.
